onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -337,6 +337,13 @@ def _permute_input_wrangler(
     return args, kwargs
 
 
+def _scatter_add_input_wrangler(
+    args: list[Any], kwargs: dict[str, Any]
+) -> tuple[list[Any], dict[str, Any]]:
+    kwargs["dim"] = args.pop(1)
```

Comment:
Curious why this is needed?

Reply:
The original test case is:  self(tensor), dim( [int, int, ...]), index(tensor), src(tensor). 
so the dim in the second place, cannot not be parsed.
if we use wrangler to change it to Tensor, the follow op.ScatterElements(self, index, src, axis=dim) will be failed, due to dim is not an attribute.